discord/webhook.py

```diff
@@ -0,0 +1,119 @@
+from .embeds import Embed
+from .mixins import Hashable
+from .errors import HTTPException
+from .utils import _bytes_to_base64_data
+import aiohttp
+import io
+import json
+import asyncio
+
+# Webhook code made by Cerulean#7014 
+
+WEBHOOK_BASE = 'https://discordapp.com/api/webhooks/'
+AVATAR_BASE = 'https://cdn.discordapp.com/avatars/{id}/{avatar}.png'
+class Webhook(Hashable):
+    __slots__ = ( 'name', 'id', 'avatar', 'url', '_state',
+                 'webhook', 'token')
+    def __init__(self,state, **kwargs):
+        self._state = state
+        
+        self.name = kwargs.get('name',None)
+        self.id = kwargs.get('id',0)
+        self.token = kwargs.get('token','')
+        self.url = WEBHOOK_BASE + '{0.id}/{0.token}'.format(self)
+        self.avatar = AVATAR_BASE.format(id=self.id,

```

Comment:
Nowhere in the library is `avatar` used to refer to the CDN. This is reserved for `avatar_url`.